### DISCUSSION TITLE
Newbie

### DISCUSSION THREAD

**Discussion by YHTxFamilia** [community/community#36199] [New to GitHub]
Hi sir ..
Im newbie 😅😅
What is it ? and can i learning ?? 😅😅
Because Its look so fun 😄

**Comment by DSDmark** [comment]
Hey @YHTxFamilia,

> Nice to meet you and welcome to this great community, we are here for helping each other and try to make **open source** better.
This is the discussions section, if you want or need anything you can ask here, we try to help.

**Comment by jfullstackdev** [comment]
you mean coding ? 

you need code to build systems, be it a website or a server side service, electronics etc., as long as it is programmable,

what's the power of coding ? well we are living in the modern world because of this, imagine a world without websites and business solutions, we will be doing them manually, say, a very fast-paced business transaction that will impact its operation and instead of a profit, it might be a loss to businesses

even cars is becoming digital as we are seeing development of electronic cars,

so, yes, that's the power of computing, it is always related to the modern world that enabled us to do things that were not possible in the past

**Comment by HalemoGPA** [comment]
### **Github** 
 in short explanation is a place where you can put all your work in, organize it
 and work in teams to enhance these projects. The place where you put your projects or works is called 
 repositiory. 
 **To** work efficiently on **github** you have to learn about git. There are many courses about **Git & Github** online.

**Comment by ossanna16** [comment]
👋 Welcome to the GitHub Community on Discussions, @YHTxFamilia! Happy to have you here 🙂
To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204). To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).
